limits/slp-0002.md

```diff
@@ -0,0 +1,44 @@
+## Preamble
+
+```
+SLP: 0002
+Title: Increase Ledger Wide Read Bytes Limit for Soroban 
+Authors:  Tomer Weller <@tomerweller>
+Consulted:
+Status: Draft
+Created: 2025-02-06
+```
+
+## Summary
+
+This proposal suggests increasing the ledger-wide read byte limit which is a bottleneck for existing protocols. 
+
+## Goals Alignment
+
+Increasing the limit will support existing protocols to operate at acale, faciliating efficient and scalable everyday financial services.
+
+## Limit Change Rationale
+
+### Ledger read bytes limit
+
+Soroban is often in surge pricing (20%-40% of Ledgers, Early February 2025) and the only resource getting exhausted is the read bytes.
+
+Examining a simple [SoroSwap invocation](https://stellar.expert/explorer/public/tx/238820179026497536#238820179026497537) reveals that 64kb are read, mostly due to reading WASM (two contracts in this instance). With the current ledger limit of 500kb this amounts to a theoretical limit of 7 swaps per ledger. Other ledger wide limits (read entries, write entries, instructions, ledger write bytes) allow for more than 50 similar swaps per ledger.  
```

Comment:
> mostly due to reading WASM (two contracts in this instance)

isn't this significantly improved after the module cache CAP gets implemented?

Reply:
Yeah the proposed caching will completely negate this issue. However, We don't have a P23 mainnet date yet and in the mean while we're surge pricing on less than 1.5 simple swaps per second